service/chain/disk.go

```diff
@@ -185,6 +185,42 @@ func (d *Disk) Results(height uint64) ([]*flow.TransactionResult, error) {
 	return converted, nil
 }
 
+func (d *Disk) Seals(height uint64) ([]*flow.Seal, error) {
+
+	blockID, err := d.block(height)
+	if err != nil {
+		return nil, fmt.Errorf("could not get block for height: %w", err)
+	}
+
+	// LookupPayloadSeals() returns the IDs of all the seals in the specified block.
+	// It should not be confused with LookupBlockSeal(), which returns the ID of the
+	// *last* payload seal found in the block.
+
+	var sealIDs []flow.Identifier
+	err = d.db.View(operation.LookupPayloadSeals(blockID, &sealIDs))
+	if err != nil {
+		return nil, fmt.Errorf("could not retrieve seal IDs: %w", err)
+	}
+
+	if len(sealIDs) == 0 {
+		return nil, nil
+	}
+
+	seals := make([]*flow.Seal, 0, len(sealIDs))
+	for _, s := range sealIDs {
+
+		var seal flow.Seal
+		err = d.db.View(operation.RetrieveSeal(s, &seal))
+		if err != nil {
+			return nil, fmt.Errorf("could not retrieve seal: %w", err)
+		}
+
+		seals = append(seals, &seal)
+	}
```

Comment:
```suggestion
	seals := make([]*flow.Seal, 0, len(sealIDs))
	for _, s := range sealIDs {
		var seal flow.Seal
		err = d.db.View(operation.RetrieveSeal(s, &seal))
		if err != nil {
			return nil, fmt.Errorf("could not retrieve seal: %w", err)
		}

		seals = append(seals, &seal)
	}
```

Reply:
You guys seriously hate newlines 😄
I hate that Go formatter reduces to two consecutive newlines at a max